test/stubs/connectors/StubIFSConnector.scala

```diff
@@ -250,6 +250,7 @@ object StubIFSConnector {
       sequence = Option(2),
       submissionDate = LocalDateTime.now())
   val api1870EmptyResponse: api_1870.SuccessResponseSchema = api_1870.SuccessResponseSchema(List.empty)
+  val api2085EmptyResponse: api_2085.ListOfIncomeSources   = api_2085.ListOfIncomeSources(List.empty)
```

Comment:
This should be in StubIFSBusinessDetailsConnector instead (I realise a lot of the empty vals in here should also be in the other stub connector as well)